src/pmi/PMIDriver.cs

```diff
@@ -194,6 +194,11 @@ private static PrepAllResult Compute(PrepAllInfo pi)
 
                 Console.WriteLine($"DRIVEALL: Invoking {driverName} {newCommandLine}");
 
+                foreach (var pair in pi.environment)
+                {
+                    Console.WriteLine($"    {pair.Key}={pair.Value}");
```

Comment:
Isn't this going to be pretty noisy?

Reply:
Its going to print around 12-15 lines per `.log` file. Compared to the `diff` file this isn't a lot, but it might be noisier than desired.

> In your particular case above, are you running PMI asm diffs as part of jit-diff? There's some magic happening there that simply outputting the environment variables won't help identify, namely, it copies the base or diff JIT into the Core_Root directory.

Yes, this is `jit-diff.bat diff --diff --pmi`. 

If there is a better/easier way to do that, I'd be all ears 😄. But as of now, printing the failing method number and these environment variables allows me to do something similar to:
```
$env:COMPlus_JitDisasm="*"
$env:COMPlus_JitDisasmAssemblies="System.Private.CoreLib"
$env:COMPlus_JitUnwindDump="*"
$env:COMPlus_JitEHDump="*"
$env:COMPlus_JitDiffableDasm=1
$env:COMPlus_ReadyToRun=0
$env:COMPlus_ZapDisable=1
$env:COMPlus_JitEnableNoWayAssert=1
$env:COMPlus_JitNoForceFallback=1
$env:COMPlus_JitRequired=1
$env:COMPlus_TieredCompilation=0
$env:COMPlus_JitStdOutFile="D:\Users\tagoo\Source\repos\runtime_base\artifacts\diffs\dasmset_5\diff\System.Private.CoreLib.dasm" 

copy D:\Users\tagoo\Source\repos\runtime\artifacts\tests\coreclr\windows.x64.Debug\Tests\Core_Root\clrjit.dll D:\Users\tagoo\Source\repos\runtime\artifacts\tests\coreclr\windows.x64.Debug\Tests\Core_Root

devenv /debugexe D:\Users\tagoo\Source\repos\runtime\artifacts\tests\coreclr\windows.x64.Release\Tests\Core_Root\corerun.exe D:\Users\tagoo\Source\repos\jitutils\bin\pmi.dll PREPONE D:\Users\tagoo\Source\repos\runtime_base\artifacts\tests\coreclr\windows.x64.Release\Tests\Core_Root\System.Private.CoreLib.dll #####
```

(Setting `AltJit` to the debug clrjit.dll also works)

https://github.com/dotnet/jitutils/issues/326 is also slightly problematic as you have to debug to see certain asserts.